src/components/views/settings/tabs/user/PreferencesUserSettingsTab.js

```diff
@@ -116,6 +136,13 @@ export default class PreferencesUserSettingsTab extends React.Component {
                                                      label={_t('Start automatically after system login')} />;
         }
 
+        let autoHideMenuOption = null;
+        if (this.state.alwaysShowMenuBarSupported) {
+            autoHideMenuOption = <LabelledToggleSwitch value={this.state.alwaysShowMenuBar}
+                                                       onChange={this._onAlwaysShowMenuBarChange}
```

Comment:
it's badly demonstrated by this file, but we only indent by 4 not until the things line up.

would recommend:
```
autoHideMenuOption = (
    <LabelledToggleSwitch value={this.state.alwaysShowManuBar}
        onChange=... etc
    />
);
```

Reply:
I need to fix my IDE config to auto this for me :((((

Reply:
WebStorm's default settings do this if you don't include the first attribute on the first line btw:

```
thing = (
    <Tag
        attr="hello world"
    />
);
```